<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dispWindow</class>
 <widget class="QMainWindow" name="dispWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dispWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>361</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Online</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Машины в сети</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>351</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="Activ_Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>17</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Последняя активность (Включая активные)</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="activDataTime">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>194</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="activeCarQTable">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>83</y>
         <width>341</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>351</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>Подробно</string>
      </property>
      <widget class="QTabWidget" name="tabWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>351</width>
         <height>361</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>Экипаж</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_10">
        <attribute name="title">
         <string>Датчики</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_11">
        <attribute name="title">
         <string>Рассылки</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Отчеты</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Поиск пройденого маршрута</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>144</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Сбросить таблицу</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>114</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Показания с датчиков</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>53</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>83</y>
         <width>194</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>57</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Начало  периода:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>87</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Конец периода:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>114</y>
         <width>141</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать маршрут</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>144</y>
         <width>141</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Сбросить маршрут</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>33</y>
         <width>61</width>
         <height>10</height>
        </rect>
       </property>
       <property name="text">
        <string>Экипаж </string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>23</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>361</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>Показания датчиков</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>351</width>
         <height>471</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Рассылка</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Сообщение</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Отправленые</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>351</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Подробно</string>
      </property>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>300</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>300</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Лист рассылок</string>
      </property>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>351</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Состояние системы</string>
     </attribute>
     <widget class="Marble::MarbleWidget" name="MarbleWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>671</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The Earth</string>
      </property>
      <property name="whatsThis">
       <string>The Marble widget displays a virtual globe.</string>
      </property>
      <property name="zoom">
       <number>2100</number>
      </property>
      <property name="mapThemeId">
       <string>earth/openstreetmap/openstreetmap.dgml</string>
      </property>
      <property name="longitude" stdset="0">
       <double>82.615255000000005</double>
      </property>
      <property name="latitude" stdset="0">
       <double>49.905400000000007</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Редоктирование геозон</string>
     </attribute>
     <widget class="Marble::MarbleWidget" name="MarbleWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>671</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The Earth</string>
      </property>
      <property name="whatsThis">
       <string>The Marble widget displays a virtual globe.</string>
      </property>
      <property name="mapThemeId">
       <string>earth/openstreetmap/openstreetmap.dgml</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Отчеты</string>
     </attribute>
     <widget class="Marble::MarbleWidget" name="MarbleWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>491</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The Earth</string>
      </property>
      <property name="whatsThis">
       <string>The Marble widget displays a virtual globe.</string>
      </property>
      <property name="mapThemeId">
       <string>earth/openstreetmap/openstreetmap.dgml</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>921</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Построение графика</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>54</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Начало  периода:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>84</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Конец периода:</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>30</y>
         <width>61</width>
         <height>10</height>
        </rect>
       </property>
       <property name="text">
        <string>Экипаж </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Датчик</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>191</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>0</y>
         <width>581</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>График</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Редактирование экипажей</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>921</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>901</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>411</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Информация об экипаже</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_12">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>411</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Настройки датчиков</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Marble::MarbleWidget</class>
   <extends>QWidget</extends>
   <header>MarbleWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
